lib/rubocop/cop/rails/date.rb

```diff
@@ -109,6 +109,16 @@ def safe_chain?(chain)
           (chain & bad_methods).empty? || !(chain & good_methods).empty?
         end
 
+        def safe_to_time?(node)
+          receiver, _method_name, *args = *node
+          if receiver.str_type?
+            zone_regexp = /[\+-]{1}[\d:]+$/
```

Comment:
Matching exactly one sis the default behaviour, so you shouldn't need the `{1}` quantifier. 😀 Also `+` does not need to be escaped in a list, so the whole thing could be simplified to:

```
[+-][\d:]+$
```

(My personal preference is for regular expressions to be stored in constants, but I don't think there's a convention for this in the project.)